web/styles/settings.css

```diff
@@ -336,6 +336,14 @@ select.settings_select {
     word-break: normal;
 }
 
+.last_active {
+    .loading_indicator_spinner {
+        width: 38px;
+        height: 20px;
```

Comment:
Height is already set to `20px` for indicators in settings section, so can this be removed?

Also, have you checked how this looks for different font sizes?